packages/node-cli/src/utils/ApplyContentProtectionUtils.js

```diff
@@ -0,0 +1,36 @@
+/*
+ ** Copyright (c) 2020 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+
+'use strict';
+
+const NodeTranslationService = require('../services/NodeTranslationService');
+const ProjectInfoService = require('../services/ProjectInfoService');
+const { LINKS, PROJECT_ACP } = require('../ApplicationConstants');
+
+const APPLY_CONTENT_PROTECTION = 'applycontentprotection';
+
+const {
+	UTILS: {
+		APPLY_CONTENT_PROTECTION_ARGUMENT_HANDLER: { ERRORS },
+	},
+} = require('../services/TranslationKeys');
+
+function validate(args, projectFolder, commandName) {
+	let projectInfoService = new ProjectInfoService(projectFolder);
+	if (args[APPLY_CONTENT_PROTECTION] && projectInfoService.getProjectType() === PROJECT_ACP) {
+		throw NodeTranslationService.getMessage(ERRORS.APPLY_CONTENT_PROTECTION_IN_ACP);
+	}
+
+	if (args[APPLY_CONTENT_PROTECTION] && !this._projectInfoService.hasLockAndHideFiles()) {
```

Comment:
I think you wanted to use projectInfoService instead of this._projectInfoService